### ISSUE TITLE
searching accuracy is low

### ISSUE THREAD

**Issue by diligent321** [liuzechun/MetaPruning#35]
Hi zechun,

I'm very interested in this work. However, when i follow the searching procedure, the accuracy on subval5w is only 3.7%. It seems unreasonable, but i can not find the reason. My steps are as follows,

step1. split the imagenet train data to subtrain and subval5w, and randomly select 2w samples from subtrain data.
step2. train the mobilenetV2 pruningnet on subtrain data, after 64 epochs, the train Acc@1 is about 45%.
step3. load the pretrain model, set subtrain2w and subval5w as train and val data respectively, and run the search procedure. I double checked the loaded model, it's right the pretrained one.
step4. the Acc@1 on subval5w is about 3.7%, the log is as follows,

"""
loaded checkpoint /data/glusterfs_workspaces/11070468/MetaPruning-master-P40-batchSize/searching_log/mobilenetV2/models/checkpoint.pth.tar epoch = 63
population_num = 50 select_num = 50 mutation_num = 25 crossover_num = 25 random_num = 0 max_iters = 20
random select ........
random_num = 50
test 1th model
[18, 23, 18, 18, 28, 28, 28, 27, 27, 27, 27, 2, 2, 2, 12, 12, 12, 15, 4, 12, 25, 23, 20, 14, 24, 8, 19, 26, 2, 10, 24, 16, 6, 26, 10]
FLOPs = 207.98M
(18.0, 23.0, 18.0, 18.0, 28.0, 28.0, 28.0, 27.0, 27.0, 27.0, 27.0, 2.0, 2.0, 2.0, 12.0, 12.0, 12.0, 15.0, 4.0, 12.0, 25.0, 23.0, 20.0, 14.0, 24.0, 8.0, 19.0, 26.0, 2.0, 10.0, 24.0, 16.0, 6.0, 26.0, 10.0)
train batch 0, loss: 2.62265944480896
train batch 1, loss: 2.5418784618377686
train batch 2, loss: 2.592099666595459
train batch 3, loss: 2.5686259269714355
train batch 4, loss: 2.498969316482544
train batch 5, loss: 2.5880398750305176
train batch 6, loss: 2.5917248725891113
train batch 7, loss: 2.5251238346099854
train batch 8, loss: 2.6498796939849854
train batch 9, loss: 2.642388343811035
train batch 10, loss: 2.4585468769073486
train batch 11, loss: 2.495739221572876
train batch 12, loss: 2.482715606689453
train batch 13, loss: 2.5206358432769775
train batch 14, loss: 2.7063348293304443
train batch 15, loss: 2.5236918926239014
train batch 16, loss: 2.5139198303222656
train batch 17, loss: 2.490814208984375
train batch 18, loss: 2.5737462043762207
train batch 19, loss: 2.40185809135437
eval batch 0, loss: 7.643044948577881
eval batch 1, loss: 8.309141159057617
eval batch 2, loss: 9.017401695251465
eval batch 3, loss: 7.960979461669922
eval batch 4, loss: 7.350314617156982
eval batch 5, loss: 8.273918151855469
eval batch 6, loss: 8.442761421203613
eval batch 7, loss: 7.337158203125
eval batch 8, loss: 7.1714630126953125
eval batch 9, loss: 8.070206642150879
eval batch 10, loss: 8.065461158752441
eval batch 11, loss: 8.770538330078125
eval batch 12, loss: 7.867849349975586
eval batch 13, loss: 8.286103248596191
eval batch 14, loss: 7.584053039550781
eval batch 15, loss: 6.936843395233154
eval batch 16, loss: 6.62252140045166
eval batch 17, loss: 8.285584449768066
eval batch 18, loss: 7.869260787963867
eval batch 19, loss: 7.220644474029541
eval batch 20, loss: 7.309091567993164
eval batch 21, loss: 7.2295942306518555
eval batch 22, loss: 7.227307319641113
eval batch 23, loss: 7.233340740203857
eval batch 24, loss: 7.1270551681518555
eval batch 25, loss: 6.655908584594727
eval batch 26, loss: 7.097906112670898
eval batch 27, loss: 6.717674255371094
eval batch 28, loss: 6.4711737632751465
eval batch 29, loss: 6.653423309326172
eval batch 30, loss: 6.54898738861084
eval batch 31, loss: 6.594447612762451
eval batch 32, loss: 6.689351558685303
eval batch 33, loss: 7.512929916381836
eval batch 34, loss: 7.239442825317383
eval batch 35, loss: 7.165963172912598
eval batch 36, loss: 6.749525547027588
eval batch 37, loss: 6.8322224617004395
eval batch 38, loss: 7.071037769317627
eval batch 39, loss: 6.838944435119629
eval batch 40, loss: 6.563052654266357
eval batch 41, loss: 6.997549533843994
eval batch 42, loss: 6.260917663574219
eval batch 43, loss: 6.890049934387207
eval batch 44, loss: 7.117074489593506
eval batch 45, loss: 6.359354496002197
eval batch 46, loss: 8.009349822998047
eval batch 47, loss: 6.258756637573242
eval batch 48, loss: 7.236196994781494
eval batch 49, loss: 7.505835056304932
 * Acc@1 3.722 Acc@5 10.550
Top1_err = 96.28 Top5_err = 89.45 loss = 7.3050
test 2th model
[14, 4, 1, 1, 17, 17, 17, 4, 4, 4, 4, 6, 6, 6, 14, 14, 14, 8, 16, 19, 24, 20, 18, 26, 26, 2, 7, 30, 30, 27, 0, 8, 6, 12, 14]
FLOPs = 133.81M
"""

looking forward to your suggestions. @liuzechun

**Comment by liuzechun0216** [comment]
Hi, 
Thanks a lot for your intersting in our work! Sorry for my late reply. 
Did you retrain the BatchNorm in search time?

**Comment by liuzechun** [comment]
Hi,
Thanks a lot for your intersting in our work! Sorry for my late reply.
Did you retrain the BatchNorm in search time?
